src/log.rs

```diff
@@ -368,7 +368,6 @@ struct Reading {
 enum ReadingState {
 	Reading,
 	Idle,
-	Shutdown,
```

Comment:
`Shutdown` is not actually needed. Log worker thread makes sure that the log is read till the end before exiting, so we don't leave the reader halfway done.